examples/move/scratch_off/sources/math.move

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module scratch_off::math {
+
+    /// Hard coding this as 1 billion
+    const MODULO_FACTOR: u256 = 1_000_000_000;
+
+    public fun bytes_to_u256(hashed_beacon: &vector<u8>): u256 {
+        let output = 0;
+        let bytes_length = 32;
+        let idx = 0;
+        while (idx < bytes_length) {
+            let current_byte = *std::vector::borrow(hashed_beacon, idx);
+            output = (output << 8) | (current_byte as u256) ;
+            idx = idx + 1;
+        };
+        output
+    }
+
+    public fun get_result(random_number: u256): u64 {
+        ((random_number % MODULO_FACTOR) as u64)
+    }
+
+    public fun get_random_u64(hashed_beacon: &vector<u8>): u64 {
+        let output: u256 = 0;
+        let bytes_length: u64 = 32;
+        let idx: u64 = 0;
+        while (idx < bytes_length) {
+            let current_byte = *std::vector::borrow(hashed_beacon, idx);
+            output = (output << 8) | (current_byte as u256) ;
+            idx = idx + 1;
+        };
+        ((output % MODULO_FACTOR) as u64)
+    }
+
+    public fun get_random_u64_in_range(hashed_beacon: &vector<u8>, max_range: u64): u64 {
+        let output: u256 = 0;
```

Comment:
Same here - this is a combination of get_random_u64 and a modulus operation, right?

Also, might be worth adding a comment that max_range shouldn't be too large (larger than 48 bits or so) to avoid a non-negligible bias in the output.